web/src/components/Failure.tsx

```diff
@@ -23,7 +23,7 @@ type FailureProps = {
   errorMessage: string
   goToPath?: string
   goToMessage?: string
-  t: TFunction<'error'>
+  t: TFunction
```

Comment:
I think this is a little dangerous, I think we should just not pass it as a prop but instead use the hook again.